test/tests/imageserver/test_microkernel_image_service.py

```diff
@@ -0,0 +1,152 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+his test validate the image-service of RackHD. It stores the micorkernel image used for node discovery.
+This test focus on image service microkernel related functionality itself.
+'''
+
+import os
+import sys
+import urllib2
+import json
+import requests
+import flogging
+import fit_common
+import unittest
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+try:
+    MICROKERNEL_CONFIG = json.loads(open(fit_common.CONFIG_PATH + "fileserver_config.json").read())
+except BaseException:
+    logs.debug_3(
+        "**** Global Config file: " + fit_common.CONFIG_PATH + "fileserver_config.json" +
+        " missing or corrupted! Exiting....")
+    sys.exit(255)
+test_microkernel_url = MICROKERNEL_CONFIG["microkernel"]
+
+
+@attr(all=True, regression=False, smoke=False)

```

Comment:
add a group here for imageservice=True (or something good).
That way we can set up a jenkins job to target this suite specifically.